packages/perseus-core/src/parse-perseus-json/regression-tests/item-data/item-with-null-itemDataVersion.json

```diff
@@ -0,0 +1,111 @@
 practicing with me. I couldn’t disappoint him. “No, I’m here to try out. Er…that is…for football, I mean. For quarterback.” Smooth. So smooth.\n\n12. Raj laughed, Marcus grinned, and Coach’s mouth twisted like I'd just force-fed him a lemon \n\n13. “Well, you see,” he started, “It’s just that girls aren’t allowed on the football team. Too dangerous, you know.”\n\n14. *But it's not too dangerous for boys? What’s wrong with people?*\n\n15. “Actually, sir, that’s not true. There’s nothing saying we can’t play.” I pulled out my copy of the school handbook. “See?” I said. “The only qualifications are that a player must have at least a C average and that the coach and a doctor must declare the player physically fit. I have straight As, and here’s my doctor’s note.”\n\n16. He grimaced, and then his face lit up. “Well, I don’t declare you physically fit. So that’s that.” He folded his arms across his chest with a satisfied grin.\n\n17. “Why?” I asked.\n\n18. “Girls are too delicate for football. I don't care how good your brother was. You'll snap like a twig out there. Sorry.”\n\n19. I’d come prepared with a second printout. Silently, I handed the paper to Coach. It contained only 37 words.\n\n20. “What in the name of the [[☃ definition 1]] is this?” he asked.\n\n21. “A copy of Title 9, sir. It’s a policy stating that any school that receives money from the government–which ours does–can’t discriminate on the basis of gender. So, I think your reasoning might be invalid. Can we get on with tryouts now?”\n\n22. His face contorted with rage, then confusion, and finally resignation as he pointed towards the center line. “You wanna be quarterback? Show me you can get past these three–” he pointed to Marcus, Raj, and Hiro, “and get the ball to the end zone.” He tossed me a ball.\n\n23. I gulped. I’d expected we’d start slow, maybe with some sprints. \n\n24. *You’ve got this,* I coached myself as I walked toward the center line. *You’re ready. Even Malik said so.* \n\n25. When I arrived at the center line and saw the three boys between me and the end zone, a sense of calm settled over me. Strategy flooded my mind as I thought of everything I’d observed from the bushes. Marcus, on the right, was big but slow. Raj, in the center, had a tendency to cover the left side of his area. Hiro, on the left, was the biggest threat–he was fast and clever, and, as he smirked at me, I knew immediately what my best play would be.\n\n26. As Coach blew his whistle, I raced hard towards the left, ignoring the obvious opening on the right side of the field. No one who knew him ever tried to get past Hiro, so that’s exactly what I would do. I barely saw Hiro’s smirk change into a look of surprise as I approached him, sliding under his outstretched arms. I leapt into the end zone with a whoop of delight. \n\n27. Coach looked reluctantly impressed when I trotted back to him. “Well, that wasn’t half-bad. I think we might give you a shot after all.”\n\n[[☃ explanation 1]]\n\n\n=====",
```

Comment:
I don't really love how much noise we get from using full content. It feels a little counter-productive because I'm not sure what parts of this are actually important to the PR. Maybe it'll make more sense when we start making changes to the snapshots, but right now it just feels like a distraction.